README.md

```diff
@@ -6,13 +6,9 @@ before spawning Ruby processes.
 
 ## Installation
 
-To install rbenv-vars, clone this repository into your
-`~/.rbenv/plugins` directory. (You'll need a recent version of rbenv
-that supports plugin bundles.)
+Make sure you have the latest version of rbenv, then run:
 
-    $ mkdir -p ~/.rbenv/plugins
-    $ cd ~/.rbenv/plugins
-    $ git clone https://github.com/rbenv/rbenv-vars.git
+    git clone https://github.com/rbenv/rbenv-vars.git $(rbenv root)/plugins/rbenv-vars
```

Comment:
Thanks @mislav but `git clone` will take care of creating what it calls the "leading directories", so there's no need to explicitly create the `plugins/` directory beforehand:

```shell
$ pwd
/tmp/blegga
$  tree .
.

0 directories, 0 files
$ git clone https://github.com/rbenv/rbenv-vars.git ${PWD}/foo/bar/qux/plugins/rbenv-vars
Cloning into '/tmp/blegga/foo/bar/qux/plugins/rbenv-vars'...
remote: Counting objects: 90, done.
remote: Total 90 (delta 0), reused 0 (delta 0), pack-reused 90
Unpacking objects: 100% (90/90), done.
Checking connectivity... done.
Running 'git submodule update' from post-checkout hook...
$ tree .
.
└── foo
    └── bar
        └── qux
            └── plugins
                └── rbenv-vars
                    ├── LICENSE
                    ├── README.md
                    ├── bin
                    │   └── rbenv-vars
                    └── etc
                        └── rbenv.d
                            └── exec
                                └── rbenv-vars.bash

9 directories, 4 files
$ _
```